workshop1/slides/src/slides.md

```diff
@@ -675,9 +675,20 @@ main = do
 
 ## Challenges
 
+- Haskell has lots abstractions one of which is the functor abstraction. Check
```

Comment:
lost *of* abstraction <comma> one of which is…

Reply:
Fixed